src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationServiceTest.java

```diff
@@ -37,7 +37,7 @@
 
 @ActiveProfiles("integration-basic")
 @SpringBootTest
-@SuppressWarnings("PMD.TooManyMethods")
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals"})
 class ValidationServiceTest extends IntegrationBasicTestBase {
```

Comment:
This test class is fine but you are missing integration test in PublicationManagementTest class